include/z64save.h

```diff
@@ -644,8 +644,8 @@ typedef enum {
 
 #define WEEKEVENTREG_12_02 PACK_WEEKEVENTREG_FLAG(12, 0x02)
 #define WEEKEVENTREG_12_04 PACK_WEEKEVENTREG_FLAG(12, 0x04)
-#define WEEKEVENTREG_12_08 PACK_WEEKEVENTREG_FLAG(12, 0x08)
-#define WEEKEVENTREG_12_10 PACK_WEEKEVENTREG_FLAG(12, 0x10)
+#define WEEKEVENTREG_SAVED_KOUME PACK_WEEKEVENTREG_FLAG(12, 0x08)
+#define WEEKEVENTREG_RECEIVED_RED_POTION_BOTTLE PACK_WEEKEVENTREG_FLAG(12, 0x10)
```

Comment:
I wonder if this should be named after the source instead of the content? Like `WEEKEVENTREG_RECEIVED_KOTAKE_BOTTLE`? Thoughts?